### PR TITLE
Solution of feed animals

### PR DESCRIPTION
**Pull request by svyshcho**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1 +1,51 @@
...
+            return self.appetite
+        return 0
+
+
+class Cat(Animal):
+    def __init__(self,
+                 name: str,
+                 appetite: int = 3,
+                 is_hungry: bool = True) -> None:
+        super(Cat, self).__init__(name, appetite, is_hungry)
```
Comment (olena-hul) [inline app/main.py:27]:
```suggestion
        super().__init__(name, appetite, is_hungry)
```

File: app/main.py [hunk 2]
```diff
@@ -1 +1,51 @@
...
+    def catch_mouse() -> None:
+        print("The hunt began!")
+
+
+class Dog(Animal):
+    def __init__(self,
+                 name: str,
+                 appetite: int = 7,
+                 is_hungry: bool = True) -> None:
+        super(Dog, self).__init__(name, appetite, is_hungry)
```
Comment (olena-hul) [inline app/main.py:39]:
```suggestion
        super().__init__(name, appetite, is_hungry)
```

**Review by olena-hul — CHANGES_REQUESTED** [review]

File: app/main.py (hunk 1 above)
Comment (svyshcho) [reply app/main.py:27]: Done

File: app/main.py [hunk 3]
```diff
@@ -1 +1,47 @@
...
+                 is_hungry: bool = True) -> None:
+        super().__init__(name, appetite, is_hungry)
+
+    @staticmethod
+    def bring_slippers() -> None:
+        print("The slippers delivered!")
+
+
+def feed_animals(animals: list[Animal]) -> int:
+    return sum(i.feed() for i in animals if i.is_hungry)
```
Comment (Bohdan-Salamakha) [inline app/main.py:47]: what is "i"?
"animal" will be more informative

**Review by Bohdan-Salamakha — CHANGES_REQUESTED** [review]
v1

File: app/main.py (hunk 3 above)
Comment (svyshcho) [reply app/main.py]: Done

**Pull request closed without merging**